### PR TITLE
Add sidebar toggle for mobile, add mobile-nav, change test screensize

### PR DESCRIPTION
**Pull request by coleerikson**
### What changed, and why?
Adds a mobile navbar to allow toggling the sidebar nav for mobile.

### How will this affect user permissions?
- These enabled user actions via mobile.

### How is this tested? (please write tests!) 💖💪
It's not in this branch. It's just mobile styles. Writing tests after.

### Screenshots please :)

Screen Shot 2020-09-24 at 10 22 34 AM

Screen Shot 2020-09-24 at 10 22 41 AM

### CODE REVIEW

File: app/javascript/src/sidebar.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  $('#sidebar-js').on("click", function() {
+    const isOpen = $( "#sidebar-js" ).hasClass( "sidebar-open" );
+    if (isOpen) {
+      $('#sidebar-js').removeClass('sidebar-open');
+    } else {
+      $('#sidebar-js').addClass('sidebar-open');
+    }
+  });
+})
```
Comment (drborges) [inline app/javascript/src/sidebar.js:20]: Small DRY up:

```js
/* global $ */
function toggleSidebar() {
  const isOpen = $( "#sidebar-js" ).hasClass( "sidebar-open" );
  if (isOpen) {
    $('#sidebar-js').removeClass('sidebar-open');
  } else {
    $('#sidebar-js').addClass('sidebar-open');
  }
}

$('document').ready(() => {
  $('#toggle-sidebar-js, #sidebar-js').on("click", toggleSidebar);
})
```

File: app/views/dashboard/_supervisors_table.html.erb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<% if policy(:dashboard).see_supervisors_section? %>
```
Comment (drborges) [inline app/views/dashboard/_supervisors_table.html.erb:1]: FYI, this partial was removed so you may run into conflicts once you pull down `new-layout`.

File: spec/support/capybara.rb [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 end
 
 Capybara.register_driver :selenium_chrome_headless_in_container do |app|
   Capybara::Selenium::Driver.new app,
     browser: :remote,
-    url: "http://selenium_chrome:4444/wd/hub",
+    url: 'http://selenium_chrome:4444/wd/hub',
     desired_capabilities: Selenium::WebDriver::Remote::Capabilities.chrome(
-      chromeOptions: {args: %w[headless disable-gpu]}
+      chromeOptions: { args: %w[headless disable-gpu --window-size=1280,900] }
```
Comment (drborges) [inline spec/support/capybara.rb:16]: 👌

**Review by drborges — APPROVED** [review]
This looks great! Do you think making the mobile menu fullscreen could look nicer?

**Comment by compwron** [comment]
You are invited to the CASA asynchronous retrospective-and-futurespective! Please take a look at https://bit.ly/casaretro (no login required) and add at least one note! Even if it's just "I was here and you are cool" or "omg y'all why do you keep messaging me" we want to hear it. <3
Next week Wednesday 18 November @ 6pm Pacific time we will Zoom about the board and talk about themes and takeaways. (Zoom link in our calendar at http://bit.ly/casacal or slack #casa channel https://rubyforgood.herokuapp.com/ )
Please ESPECIALLY add lots of notes if you won't be able to make it on Wednesday to talk!
* Note: the board will be monitored for spam; message me or any team lead if you see a problem
https://bit.ly/casaretro

**Pull request merged**